00:40:14 - main - WARNING - <module>: 18 - ________________________________HELLO__________________________________
00:43:20 - main - WARNING - <module>: 18 - ________________________________HELLO__________________________________
00:44:02 - main - WARNING - <module>: 18 - ________________________________HELLO__________________________________
00:44:03 - main - WARNING - sending: 194 - cannot write empty image as JPEG
00:44:03 - main - ERROR - <module>: 234 - cannot write empty image as JPEG
Traceback (most recent call last):
  File "/home/runner/work/FortDaily-PVE_bot/FortDaily-PVE_bot/main.py", line 191, in sending
    bot.send_photo(chat_id=channel_id, photo=page)
  File "/opt/hostedtoolcache/Python/3.9.17/x64/lib/python3.9/site-packages/telebot/__init__.py", line 1825, in send_photo
    apihelper.send_photo(
  File "/opt/hostedtoolcache/Python/3.9.17/x64/lib/python3.9/site-packages/telebot/apihelper.py", line 469, in send_photo
    files = {'photo': util.pil_image_to_file(photo)}
  File "/opt/hostedtoolcache/Python/3.9.17/x64/lib/python3.9/site-packages/telebot/service_utils.py", line 62, in pil_image_to_file
    image.convert('RGB').save(photoBuffer, extension, quality=quality)
  File "/opt/hostedtoolcache/Python/3.9.17/x64/lib/python3.9/site-packages/PIL/Image.py", line 2432, in save
    save_handler(self, fp, filename)
  File "/opt/hostedtoolcache/Python/3.9.17/x64/lib/python3.9/site-packages/PIL/JpegImagePlugin.py", line 637, in _save
    raise ValueError(msg)
ValueError: cannot write empty image as JPEG

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/runner/work/FortDaily-PVE_bot/FortDaily-PVE_bot/main.py", line 232, in <module>
    sending()
  File "/home/runner/work/FortDaily-PVE_bot/FortDaily-PVE_bot/main.py", line 201, in sending
    bot.send_photo(chat_id=channel_id, photo=page)
  File "/opt/hostedtoolcache/Python/3.9.17/x64/lib/python3.9/site-packages/telebot/__init__.py", line 1825, in send_photo
    apihelper.send_photo(
  File "/opt/hostedtoolcache/Python/3.9.17/x64/lib/python3.9/site-packages/telebot/apihelper.py", line 469, in send_photo
    files = {'photo': util.pil_image_to_file(photo)}
  File "/opt/hostedtoolcache/Python/3.9.17/x64/lib/python3.9/site-packages/telebot/service_utils.py", line 62, in pil_image_to_file
    image.convert('RGB').save(photoBuffer, extension, quality=quality)
  File "/opt/hostedtoolcache/Python/3.9.17/x64/lib/python3.9/site-packages/PIL/Image.py", line 2432, in save
    save_handler(self, fp, filename)
  File "/opt/hostedtoolcache/Python/3.9.17/x64/lib/python3.9/site-packages/PIL/JpegImagePlugin.py", line 637, in _save
    raise ValueError(msg)
ValueError: cannot write empty image as JPEG
00:48:45 - main - WARNING - <module>: 18 - ________________________________HELLO__________________________________
00:44:48 - main - WARNING - <module>: 18 - ________________________________HELLO__________________________________
00:44:10 - main - WARNING - <module>: 18 - ________________________________HELLO__________________________________
00:44:31 - main - WARNING - <module>: 18 - ________________________________HELLO__________________________________
00:43:13 - main - WARNING - <module>: 18 - ________________________________HELLO__________________________________
00:42:04 - main - WARNING - <module>: 18 - ________________________________HELLO__________________________________
00:48:59 - main - WARNING - <module>: 18 - ________________________________HELLO__________________________________
00:46:36 - main - WARNING - <module>: 18 - ________________________________HELLO__________________________________
00:44:34 - main - WARNING - <module>: 18 - ________________________________HELLO__________________________________
00:44:52 - main - WARNING - <module>: 18 - ________________________________HELLO__________________________________
00:44:49 - main - WARNING - <module>: 18 - ________________________________HELLO__________________________________
00:45:28 - main - WARNING - <module>: 18 - ________________________________HELLO__________________________________
00:48:45 - main - WARNING - <module>: 18 - ________________________________HELLO__________________________________
